test/library/packages/Python/correctness/myTuple.chpl

```diff
@@ -0,0 +1,87 @@
+use Python;
+
+var pyCode = """
+def getTup():
+  return (2, 3, 5, 7)
+
+def getTup2():
+  return (11,)
+""";
+
+proc main() {
+
+  var interp = new Interpreter();
+
+  var pyCodeModule = interp.importModule('__empty__', pyCode);
+  var getTupFunc = pyCodeModule.get('getTup');
+
+  var t = getTupFunc(owned PyTuple);
+  writeln("size ", t.size);
+  writeln("t[0] ", t.get(int, 0));
+  writeln("t[1] ", t.get(1));
+  writeln("t[2] ", t.get(int, 2));
+  writeln("t[3] ", t.get(int, 3));
+
+  // check contains
+  writeln("5 in t: ", t.contains(5));
+  writeln("8 in t: ", t.contains(8));
+
+  // get a slice of the tuple
+  writeln("t[0:2] ", t.get(0..1));
+  writeln("t[1:2] ", t.get(1..1));
+  writeln("t[2:5] ", t.get(2..5));
+  writeln("t[-3:2] ", t.get(-3..1));
+  writeln("t[-3:5] ", t.get(-3..5));
+
+  // get a slice with an empty range
+  writeln("t[1:1] ", t.get(1..0));
+
+  // slice missing at least one bounds
+  writeln("t[2:] ", t.get(2..));
+  writeln("t[:3] ", t.get(..2));
+  writeln("t[:] ", t.get(..));
+
+  // get slice with range and type argument
+  writeln("checking explicit type for slice overload");
+  writeln("t[0:2] ", t.get(owned PyTuple, 0..1));
+  writeln("t[2:5] ", t.get(2*int, 2..5));
```

Comment:
should this work? 2*int is not enough space for the slice 2..5

Reply:
This should work because `t` only stores `0..3`.  Python allows you to specify an end bound beyond the end of the tuple, which is what we're doing here.  It just returns everything starting from the start position you specified in that case, which is two elements.

Reply:
Oh I see. Sounds good